crates/bevy_gltf/src/lib.rs

```diff
@@ -251,3 +247,90 @@ pub struct GltfExtras {
     /// Content of the extra data.
     pub value: String,
 }
+
+/// Labels that can be used to load part of a glTF
```

Comment:
Can we add a basic usage example here, or otherwise explain how this is used?

Reply:
added more docs!